index.js

```diff
@@ -76,6 +76,13 @@ module.exports = {
     this.jshintrc = app.options.jshintrc;
   },
 
+  postBuild: function () {
```

Comment:
Cool happy with both of those suggestions.

Am I right in thinking there is no current API for removing packages?